launcher/agent/agent.go

```diff
@@ -183,6 +187,16 @@ func (a *agent) AttestWithClient(ctx context.Context, opts AttestAgentOpts, clie
 		return nil, err
 	}
 
+	tokenOpts := opts.TokenOptions
```

Comment:
Can we add a test that checks for the "default" token options behavior being set here?